Update docs to highlight normalization support in configuration cache

mlopatkin opened the pull request:
Blocked on #18293

eskatos reviewed (approved):
👍

eskatos commented on subprojects/docs/src/docs/release/notes.md:
```diff
@@ -108,6 +108,10 @@ However, these events can be unreliable in some environments, which could cause
 If a file system event for this change does not arrive by the start of the next build, Gradle concludes that file system events are unreliable, and will fall back to checking the file system for files involved in the build for changes.  
 
+### Input normalization support in configuration cache
+The [input normalization](userguide/more_about_tasks.html#sec:configure_input_normalization) is now correctly tracked by the configuration cache. Task up-to-date checks now consider normalization
```
```suggestion
The [input normalization](userguide/more_about_tasks.html#sec:configure_input_normalization) is now correctly tracked by the experimental [configuration cache](userguide/configuration_cache.html). Task up-to-date checks now consider normalization
```

eskatos reviewed (changes requested):
I added a suggestion to state that the configuration cache is still experimental and add a link to the dedicated manual chapter for readers that are not aware of it yet.

mlopatkin commented:
> I added a suggestion to state that the configuration cache is still experimental and add a link to the dedicated manual chapter for readers that are not aware of it yet.

Thanks, updated

mlopatkin commented:
@bot-gradle test and merge

bot-gradle commented:
Your PR is queued. See [the queue page](https://gradle-bot.grdev.net/queue.html) for details.

bot-gradle commented:
OK, I've already triggered [a build](https://builds.gradle.org/viewQueued.html?itemId=46218437) for you.
